examples/CRM.eve

```diff
@@ -399,25 +399,25 @@ commit
 
   // Make some threads
   [#thread #new-messages users: (corey, chris) messages:
-    [#message sender: corey, time: [day month year hours minutes seconds: 0 ampm], text: "Hey"]
-    [#message sender: chris, time: [day month year hours minutes seconds: 1 ampm], text: "Hey, how are you."]
-    [#message sender: corey, time: [day month year hours minutes seconds: 2 ampm], text: "I'm fine, how are you?"]
-    [#message #unread sender: chris, time: [day month year hours minutes seconds: 3 ampm], text: "Fine as well."]
-    [#message #unread sender: chris, time: [day month year hours minutes: minutes + 1, seconds: 4 ampm], text: "Glad we got that out of the way!"]
-    [#message #unread sender: chris, time: [day month year hours minutes: minutes + 1 seconds: 5 ampm], text: "What did you work on yesterday?"]]
+    [#message sender: corey, time: [timestamp: t1, time-string], text: "Hey"]
```

Comment:
Right, we have no conversions right now though. I'm carrying around the timestamp to sort it, and the time-string to display it. Ideally, the timestamp would be carried around, and the time-string could be inferred.